tests/integration/api/controllers/replication.spec.js

```diff
@@ -38,7 +38,7 @@ const assertDocIds = (response, ...expectedIds) => {
   const receivedIds = response.doc_ids_revs
     .map(pair => pair.id)
     .filter(id => !isFormOrTranslation(id));
-  expect(receivedIds).to.include.members([ ...expectedIds, ...DEFAULT_EXPECTED ]);
+  expect(receivedIds.sort()).to.have.members([ ...expectedIds, ...DEFAULT_EXPECTED ].sort());
```

Comment:
nitpick: Do we need the `sort()` calls here?  I learned recently that `to.have.members` does a strict unordered contains assertion!  😅

Reply:
OMG this is for when your test fails and you get the differences in the log. If they are sorted, then it's super easily visible what the differences are, otherwise you have a +- mash of which element is in one array and which one in the other.

Reply:
:sweat_smile:   Ahh, got it!